resources/assets/components/Tags/index.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { map } from 'lodash';
+
+class Tags extends React.Component {
+  render() {
+
+    return (
+        <ul className="form-actions -inline">
+          <li><b> Tags </b></li>
```

Comment:
@DFurnes for some reason when I do that, it breaks everything :-/ 
```
index.js:5 Uncaught Error: Cannot find module "../Tags"
    at webpackMissingModule (index.js:5)
    at Object.<anonymous> (index.js:5)
    at __webpack_require__ (bootstrap 78da61f…:19)
    at Object.defineProperty.value (index.js:5)
    at __webpack_require__ (bootstrap 78da61f…:19)
    at Object.defineProperty.value (app.js:8)
    at __webpack_require__ (bootstrap 78da61f…:19)
    at bootstrap 78da61f…:39
    at bootstrap 78da61f…:39
    at webpackUniversalModuleDefinition (universalModuleDefinition:7)
webpackMissingModule @ index.js:5
(anonymous) @ index.js:5
__webpack_require__ @ bootstrap 78da61f…:19
Object.defineProperty.value @ index.js:5
__webpack_require__ @ bootstrap 78da61f…:19
Object.defineProperty.value @ app.js:8
__webpack_require__ @ bootstrap 78da61f…:19
(anonymous) @ bootstrap 78da61f…:39
(anonymous) @ bootstrap 78da61f…:39
webpackUniversalModuleDefinition @ universalModuleDefinition:7
(anonymous) @ universalModuleDefinition:10
```
but I'm not sure why it breaks like that?